Assets/Scripts/AI/States/AttackBehaviour.cs

```diff
@@ -0,0 +1,37 @@
+﻿using UnityEngine;
+
+public class AttackBehaviour : StateMachineBehaviour
+{
+    public GameObject particle;
+    public float radius;
+    public float power;
+
+    protected GameObject clone;
+
+    override public void OnStateEnter(Animator animator, AnimatorStateInfo stateInfo, int layerIndex)
+    {
+        clone = Instantiate(particle, animator.rootPosition, Quaternion.identity) as GameObject;
+        Rigidbody rb = clone.GetComponent<Rigidbody>();
+        rb.AddExplosionForce(power, animator.rootPosition, radius, 3.0f);
```

Comment:
Is your `particle` a gameobject or a particle system? If it is a particle system, can the movement of the particle be directly controlled by the particle system?